packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -136,10 +140,22 @@ export class AggregatedAttestationPool {
       attestationGroupByIndex = new Map<CommitteeIndex, MatchingDataAttestationGroup>();
       attestationGroupByIndexByDataHash.set(dataRootHex, attestationGroupByIndex);
     }
-    const committeeIndex = isElectraAttestation(attestation)
-      ? // this attestation is added to pool after validation
-        attestation.committeeBits.getSingleTrueBit()
-      : attestation.data.index;
+
+    let committeeIndex;
+
+    if (isForkPostElectra(this.config.getForkName(slot))) {
+      if (isElectraAttestation(attestation)) {
+        committeeIndex = attestation.committeeBits.getSingleTrueBit();
+      } else {
+        throw new Error(`Expect electra attestation but received phase0 attestation. Slot: ${slot}`);
+      }
```

Comment:
```suggestion
      if (!isElectraAttestation(attestation)) {
        throw new Error(`Expect electra attestation but received phase0 attestation. Slot: ${slot}`);
      }
      committeeIndex = attestation.committeeBits.getSingleTrueBit();
```

error message probably should also be tweaked a little to be more conistent with existing error formating, e.g. `Slot: ${slot}` is something I haven't seen so far appended at the end